### ISSUE TITLE
[Feature Request]: Support without the need of ADB

### ISSUE THREAD

**Issue by deanfourie1** [PRProd/HA-Firemote#428]
### Is your feature request related to a problem?

well kimda, I dont have ADB enabled.

### Feature Request

Support for devices when ADB is not enabled.

### Additional Comments (optional)

I have an Android TV remote entity that does not required ADB to be enabled.

Can we get this also compatible without the need of ADB?

**Comment by PRProd** [comment]
Greetings @deanfourie1!

This question/request does come up every once in a while, so I've actually addressed it in the [FAQ](https://github.com/PRProd/HA-Firemote/wiki#faq) section of the Firemote Wiki.  Take a look at the section titled [Why do I still need the ADB integration when there is an Android Remote Integration?](https://github.com/PRProd/HA-Firemote/wiki#why-do-i-still-need-the-adb-integration-when-there-is-an-android-remote-integration)

If you are unsure about how to enable ADB (USB) debugging, I've also created a YouTube Video called [Enable ADB & USB Debugging](https://www.youtube.com/watch?v=XL7neKn1d-Y) to take you through the steps of that process.

After first understanding [how Firemote sends commands to your devices](https://github.com/PRProd/HA-Firemote/wiki#how-does-firemote-send-commands-to-my-devices), if you are an advanced Home Assistant user and are opposed to using and ADB/USB debugging option on your device, you could still use the Firemote card via a feature called [entity:none](https://github.com/PRProd/HA-Firemote/wiki/entity:-none).  Keep in mind though, writing scripts or HA commands that only utilize the [Andriod TV Remote](https://www.home-assistant.io/integrations/androidtv_remote) integration will hinder your ability to do everything that your physical remote control can do.

To sum it up, as stated in the README's [Download and Setup instructions](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#download-and-setup-via-home-assistant-ui--hacs), ADB integration is a requirement, and to remove it would mean that a great many features that make Firemote what it is today would have to go away.

I hope this helps.

-Doug
